update_version.yml

```diff
@@ -0,0 +1,29 @@
+name: Update Version on Docker Files
+
+env:
+  PRJ_VER: ${{ github.event.inputs.project_version }}
+
+
+
+on:
+ workflow_dispatch:
+    inputs:
+      project_version:
+        description: 'Version number to be released e.g 10.0.3'
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: Install dependencies
+        run: sudo apt install libcurl4-openssl-dev libssl-dev
+
+      - name: Clone Tools branch
+        run: git clone --branch tools-transition-update-docker https://github.com/citusdata/tools.git
```

Comment:
I see that we need changes on different branches of tools repo. Let's make sure we change them into tags when the transition is complete